### PR TITLE
#9 create http endpoints

### PR DESCRIPTION
**Pull request by alex-a-renoire**

### CODE REVIEW

File: pkg/http/handler/handler.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	if err != nil {
+		h.reportError(w, http.StatusInternalServerError, err)
+	}
+
+	res, err := json.Marshal(items)
+	if err != nil {
+		h.reportError(w, http.StatusInternalServerError, err)
+	}
+
+	w.Header().Set("Content-Type", "application/json")
```
Comment (DrIhor) [inline pkg/http/handler/handler.go:68]: use middleware to set general format or add it to reportError()

File: pkg/http/handler/handler.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	router.Use(h.authMiddleware)
+
+	return router
+}
+
+// Returns the list of items
+func (h HTTPHandler) ListItems(w http.ResponseWriter, req *http.Request) {
+	items, err := h.ItemsService.GetItems()
+	if err != nil {
+		h.reportError(w, http.StatusInternalServerError, err)
```
Comment (DrIhor) [inline pkg/http/handler/handler.go:75]: use `return` to stop work if err part is ready and you send resp(`w.Write` inside of `reportError()`)

File: pkg/http/handler/model.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	id uuid.UUID
+}
+
+type Error struct {
+	Message string `json:"message,omitempty"`
+}
+
+func (e Error) Error() string {
+	return e.Message
+}
```
Comment (DrIhor) [inline pkg/http/handler/model.go]: Model and method may be better to separate.
Also Item can be general model if you use it as return from GRPC service.

Comment (alex-a-renoire) [reply pkg/http/handler/model.go]: So far I have separate pull requests. I will make the single model when pull-requests are merged.

**Review by DrIhor — APPROVED** [review]

**Review by romanilchyshyn — APPROVED** [review]

**Pull request merged**